Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -237,6 +237,14 @@ bool CVolumeWiiCrypted::IsWiiDisc() const
 	return true;
 }
 
+bool CVolumeWiiCrypted::IsDiscTwo() const
+{
+	bool discTwo = false;
```

Comment:
Make this a u8 and return based off of the value

ie. `return diskTwo == 1` (or whatever value signifies it's the second disk).

Casting from a bool to a pointer of a different type isn't usually a good idea.